scan

```diff
@@ -232,15 +247,16 @@ if [ $numscans -gt 0 ]; then
     if [ -f "${OUTPUT}.orig" ]; then
       pdffiles+=("${OUTPUT}.orig")
     fi
-    pdffiles+=(scan-[0-9]*.pdf)
-    pdfunite "${pdffiles[@]}" "$OUTPUT" && rm scan-[0-9]*.pdf
+    pdffiles+=($TMP_DIR/scan-[0-9]*.pdf)
+    pdfunite "${pdffiles[@]}" "$OUTPUT" && rm $TMP_DIR/scan-[0-9]*.pdf
   else
     if [ $USEARRAY = 1 ]; then
-      mv scan-0*.pdf "${OUTPUT[0]}"
+      mv $TMP_DIR/scan-0*.pdf "${OUTPUT[0]}"
     else
-      mv scan-0*.pdf "$OUTPUT"
+      mv $TMP_DIR/scan-0*.pdf "$OUTPUT"
     fi
   fi
+  rm -rf $TMP_DIR
```

Comment:
Lets use a trap to clean this up...  just like for the lockfile cleanup.

Reply:
added.